package.json

```diff
@@ -102,7 +102,6 @@
     "fuzzysearch": "1.0.3",
     "history": "5.0.0",
     "husky": "4.3.0",
-    "jest": "^26.6.3",
```

Comment:
This is provided by `react-scripts`. This was most likely added when we merged `mdn-fiori`